lib/ethdev/rte_ethdev.h

```diff
@@ -2070,6 +2072,19 @@ uint64_t rte_eth_find_next_owned_by(uint16_t port_id,
 	     (unsigned int)p < (unsigned int)RTE_MAX_ETHPORTS; \
 	     p = rte_eth_find_next_owned_by(p + 1, o))
 
+/**
+ * Returns the network namespace installed for a specific port
+ *
+ * @param port_id
+ *   The ID of the port we are interested in.
+ * @param netns
+ *   Pointer to the structure passed to fill the network namespace
+ * @return
+ *   0 if successfull,
+ *   -EINVAL if the pointer to save the namespace is null
+ */
+int rte_ethdev_netns_get(uint16_t port_id, char *netns);
```

Comment:
This is also present in rte_ethdev_netns.h
Why have it in two different headers?

Reply:
Good point. Fixed.

Reply:
if it's fixed then why do we still see it here? :)